ui/src/app/pages/analysis/confirm-playground-mode-modal.tsx

```diff
@@ -27,10 +27,9 @@ export class ConfirmPlaygroundModeModal extends React.Component<Props, State> {
     };
   }
 
-  toggleChecked() {
-    const newState = !this.state.checked;
-    this.setState({checked: newState});
-    Cookies.set(ConfirmPlaygroundModeModal.DO_NOT_SHOW_AGAIN, String(newState));
+  toggleChecked(checked: boolean) {
+    this.setState({checked: checked});
```

Comment:
shouldn't this be `checked: !checked`?

Reply:
or does this happen after the actual checked state changes? if that's the case we should change the name of this function since it is no longer toggling checked